src/PresentationalComponents/DownloadReport/DownloadReport.js

```diff
@@ -4,7 +4,7 @@ import DownloadIcon from '@patternfly/react-icons/dist/js/icons/download-icon';
 import './DownloadReport.scss';
 
 export const DownloadReport = () => (
-    <Button variant="link" icon={ <DownloadIcon /> }>
+    <Button className="ins-c-download-report__button" variant="link" icon={ <DownloadIcon /> }>
```

Comment:
`component='a'` rite?

Reply:
this should remain a button since it downloads content, the `<a>` tag is typically used when linking to another page.